mlrun/api/rundb/sqldb.py

```diff
@@ -909,3 +836,9 @@ def verify_authorization(
         # on server side authorization is done in endpoint anyway, so for server side we can "pass" on check
         # done from ingest()
         pass
+
+
+# Once this file is imported it will override the default RunDB implementation (RunDBContainer)
+@containers.override(mlrun.db.factory.RunDBContainer)
+class SQLRunDBContainer(containers.DeclarativeContainer):
+    run_db = providers.Factory(SQLRunDB)
```

Comment:
I would move this to the top of the file, as it is related to importing it.
Just to make it a bit clearer

Reply:
Can't because need to define SQLRunDB 1st